first_lessons_dushanbe/array_sorting.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+def list_sort(a):
+    if a is None:
+        return False
```

Comment:
you should not change return type depending on input conditions. Could you please make return type consistent?

Reply:
Yes, I've changed it to return a